### PR TITLE
[3] New fragment layout

### PR DESCRIPTION
**Pull request by AnuthaDev**
Added a new fragment layout with description just below title

Screenshot: https://drive.google.com/file/d/1LsiwFIRckC_m072YLTGcNrXnJnVbnLDW/view?usp=drivesdk

### CODE REVIEW

File: appintro/src/main/java/com/github/paolorotolo/appintro/internal/viewpager/ViewPagerTransformer.kt [hunk 1]
```diff
@@ -56,13 +56,16 @@ internal class ViewPagerTransformer(
             }
             TransformType.FADE -> {
                 if (position <= -1.0f || position >= 1.0f) {
+                    page.translationX = page.width.toFloat()
```
Comment (cortinico) [inline appintro/src/main/java/com/github/paolorotolo/appintro/internal/viewpager/ViewPagerTransformer.kt]: This change should not be here.
Can you please rebase?

Comment (Anutha02) [reply appintro/src/main/java/com/github/paolorotolo/appintro/internal/viewpager/ViewPagerTransformer.kt]: Me not know git😅

Comment (cortinico) [reply appintro/src/main/java/com/github/paolorotolo/appintro/internal/viewpager/ViewPagerTransformer.kt]: This is a good starting guide: http://rogerdudler.github.io/git-guide/

**Comment by Anutha02** [comment]
@cortinico please merge

**Pull request closed without merging**